2007-01-27  H.J. Lu  <hongjiu.lu@intel.com>

	* elf64-hppa.c (elf64_bed): Defined for HPUX and Linux.

--- binutils-2.17.50.0.10/bfd/elf64-hppa.c
+++ binutils-2.17.50.0.10/bfd/elf64-hppa.c
@@ -2861,6 +2861,8 @@
 #define elf_backend_action_discarded	elf_hppa_action_discarded
 #define elf_backend_section_from_phdr   elf64_hppa_section_from_phdr
 
+#define elf64_bed			elf64_hppa_hpux_bed
+
 #include "elf64-target.h"
 
 #undef TARGET_BIG_SYM
@@ -2869,9 +2871,10 @@
 #define TARGET_BIG_NAME			"elf64-hppa-linux"
 #undef ELF_OSABI
 #define ELF_OSABI			ELFOSABI_LINUX
+#undef elf64_bed
+#define elf64_bed			elf64_hppa_linux_bed
 
 #undef elf_backend_post_process_headers
 #define elf_backend_post_process_headers _bfd_elf_set_osabi
 
-#define INCLUDED_TARGET_FILE 1
 #include "elf64-target.h"
